ui-v2/app/styles/components/buttons/skin.scss

```diff
@@ -0,0 +1,90 @@
+/* decor */
+%button {
+  @extend %user-select-none;
+  cursor: pointer;
+  white-space: nowrap;
+}
+%primary-button,
+%secondary-button,
+%copy-button,
+%dangerous-button {
+  @extend %button;
+}
+%primary-button,
+%secondary-button,
+%dangerous-button {
+  border: $decor-border-100;
+  border-radius: $radius-small;
+  box-shadow: 0 3px 1px 0 rgba($ui-black, 0.12);
+}
+%primary-button,
+%copy-button {
+  border: 0;
+}
+%copy-button {
+  @extend %with-clipboard;
+}
+/* color */
+%secondary-button,
+%dangerous-button,
+%copy-button {
+  border-color: currentColor;
+}
+%secondary-button {
+  color: $ui-gray-400;
+}
+%secondary-button:hover,
+%secondary-button:focus {
+  color: $ui-gray-600;
+}
+%secondary-button:active {
+  color: $ui-gray-600;
+}
+%primary-button {
+  background-color: $ui-color-action;
+  color: $ui-white;
+}
+%primary-button:hover,
+%primary-button:focus {
+  background-color: $ui-blue-700;
+}
+%primary-button:disabled {
+  color: rgba($ui-white, 0.5);
+  background-color: rgba($ui-blue-700, 0.5);
```

Comment:
Hmm ok, this isn't really part of this PR, this is how it is in master

https://github.com/hashicorp/consul/blob/16200e4c7511a53d6a3fa6a1f5411cbc3c8de79c/ui-v2/app/styles/components/button.scss#L82-L86

It just moved to a different file.

I'm currently working with @opihana and @joshuaogle to define these things further, this will probably also be part of the `frame` idea. Again, coming to a CSS file this fall 😄

Reply:
It _is_ part of this PR if part of this PR is figuring out colors.

Reply:
I'd love to get rid of any of the `darker($color, 10%)` kind of things, but not sure we can (or should try to) get rid of the opacity styles

Reply:
Having said that, I just noticed that you've got translucent text on a translucent background here, which I would try to avoid (it's probably not an issue here but leads to weird color mixing). Would `opacity: 0.5` on the button itself be better?